src/BlueEnemy.cpp

```diff
@@ -15,10 +15,11 @@ void BlueEnemy::Draw(sf::RenderWindow& window)
 
 void BlueEnemy::Update()
 {
+    extern int WindowHeight; 
```

Comment:
Not the greatest fan of "externing" your global variables in every function you need. Generally global variables should be avoided wherever possible, but for now making screen dimensions global makes sense (maybe we will fix that in the future).

Much better option would be adding
extern int WindowWidth and extern int WindowHeight in Entity.h in the global scope? So that everything that inherits from it will be able to immediately use it without having to write it 100x times :)